tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -1294,3 +1305,113 @@ async def test_notification_rpcs(wallet_rpc_environment: WalletRpcTestEnvironmen
     notification = (await client_2.get_notifications())[0]
     assert await client_2.delete_notifications([notification.coin_id])
     assert [] == (await client_2.get_notifications([notification.coin_id]))
+
+
+@pytest.mark.parametrize(
+    "rpc_request, rpc_response",
```

Comment:
I would prefer the form that leverages Python syntax more.

```suggestion
    ["rpc_request", "rpc_response"],
```

Reply:
Updated